package-lock.json

```diff
@@ -0,0 +1,6656 @@
+{
```

Comment:
i'd recommend gitignoring this and setting `package-lock=false` in `.npmrc`; only apps should have lockfiles